portfolio/src/main/webapp/discussionScript.js

```diff
@@ -0,0 +1,8 @@
+
+function displayList(){
+  fetch('/discussion')
+  .then((response) => response.json())
+  .then((data) => {
+    document.getElementById('discussion-container').textContent = data;
```

Comment:
Remember -- the variable data here will be a list, not a string. While setting it directly to textContent will work, it might look a bit strange

See my comment here: https://github.com/kristen-m/step/pull/17
for more explanation.  Probably a good idea to use `.join(', ');` to format it better

Reply:
I have been looking at the different separators for the last 15min, and I'm not sure why the .join("\r\n") or .join("\n") dont work. Do you know why?

Reply:
Yeah so \n doesn't work as a linebreak in html, since the \ character is interpreted literally.  Instead HTML uses `<br>` tags for line break (but those are bad like we discussed before).  What you can do instead is loop through the list and create a div or li element for each word in data, then append each as a separate node.

Reply:
Okay. Whenever i ran the .join("\n"), neither the '\ ' nor the 'n' showed up in the text. It's not using it as an escape character but it seems to just consume the characters and do nothing with them?

Reply:
Oh huh interesting, looks like I was wrong about how html treats \n, seems like you can get it to show up with css.

https://stackoverflow.com/questions/39325414/line-break-in-html-with-n

I haven't seen that used very often, so I'd still recommend wrapping each line in its own element, but either way you're able to get it to work is ok!